wp-job-manager-functions.php

```diff
@@ -492,7 +507,7 @@ function wp_job_manager_notify_new_user( $user_id, $password ) {
 		global $wp_version;
 
 		if ( version_compare( $wp_version, '4.3.1', '<' ) ) {
-			wp_new_user_notification( $user_id, $password );
+			wp_new_user_notification( $user_id, $password ); // phpcs:ignore
```

Comment:
What's going on here? What are we ignoring?

Reply:
Added the explicit rule. We're skipping the error because the version check catches the change of signature in this function.

7d2a6f5f410e7e71d0d8f0bdcfc6d524b2c256b0